pkg/otlp/metrics/metrics_remapping_test.go

```diff
@@ -785,13 +785,18 @@ func TestRemapMetrics(t *testing.T) {
 			tt.in.Name() == "kafka.producer.record-retry-rate" ||
 			tt.in.Name() == "kafka.producer.record-send-rate"
 		remapMetrics(dest, tt.in)
+		// Ensure remapMetrics does not add the otel.* prefix to the metric name
 		if checkprefix {
-			require.True(t, strings.HasPrefix(tt.in.Name(), "otel."), "system.* and process.*  and a subset of kafka metrics need to be prepended with the otel.* namespace")
+			require.False(t, strings.HasPrefix(tt.in.Name(), "otel."), "system.* and process.*  and a subset of kafka metrics need to be prepended with the otel.* namespace")
```

Comment:
The test message needs to be updated too

Reply:
Here I am testing that remap_metrics no longer renames metrics (adds `otel.` prefix). This method only computes Datadog metrics. `rename_metrics` is now the function responsible for adding the `0tel.` prefix. I updates the test message to make this more clear.